openxla-nvgpu/runtime/src/openxla/runtime/nvgpu/dynamic_symbols.c

```diff
@@ -0,0 +1,76 @@
+// Copyright 2023 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "openxla/runtime/nvgpu/dynamic_symbols.h"
+
+#include <string.h>
+
+#include "iree/base/internal/dynamic_library.h"
+#include "iree/base/target_platform.h"
+#include "iree/base/tracing.h"
+
+static const char* kCuDNNLoaderSearchNames[] = {
+#if defined(IREE_PLATFORM_WINDOWS)
+    "nvcudnn.dll",
+#else
+    "libcudnn.so",
+#endif  // IREE_PLATFORM_WINDOWS
+};
+
+#define concat(A, B) A B
+
+// Load CUDA entry points, prefer _v2 version if it exists.
+static iree_status_t openxla_cudnn_dynamic_symbols_resolve_all(
+    openxla_cudnn_dynamic_symbols_t* syms) {
+#define CUDNN_PFN_DECL(cuDNNSymbolName, ...)                                  \
+  {                                                                           \
+    static const char* kName = #cuDNNSymbolName;                              \
+    IREE_RETURN_IF_ERROR(iree_dynamic_library_lookup_symbol(                  \
+        syms->cudnn_library, kName, (void**)&syms->cuDNNSymbolName));         \
+    static const char* kNameV2 = concat(#cuDNNSymbolName, "_v2");             \
```

Comment:
This was copied from the CUDA wrappers, I assume. It seems dangerous to fallback to the original when V2 was not found, because the function signature is different. Also, cuDNN's versioning scheme is different (using `_v7`/`_v8` suffixes).

Reply:
Oops, missed that, removed `_v2` thing completely.